cmd/kubeadm/app/preflight/checks.go

```diff
@@ -182,6 +184,38 @@ func (fac FileAvailableCheck) Check() (warnings, errors []error) {
 	return nil, errors
 }
 
+// FileContentCheck checks that the given file contains the string Content.
+type FileContentCheck struct {
+	Path    string
+	Content []byte
+}
+
+func (fcc FileContentCheck) Check() (warnings, errors []error) {
```

Comment:
minor nit: I think it's common practice to pass the struct as a reference, especially when accessing fields from it => `fcc *FileContentCheck`